skimage/measure/tests/test_regionprops.py

```diff
@@ -2,6 +2,8 @@
 
 import numpy as np
 from numpy import array
+
+from skimage._shared._warnings import expected_warnings
```

Comment:
What is the current opinion on this vs `pytest.warns`?

Reply:
Our warnings can be made optional

https://github.com/scikit-image/scikit-image/pull/4021

Reply:
Thanks for the pointer. I don't know the full history behind that function but was `pytest.mark.skipif(os.environ.get('SKIMAGE_TEST_STRICT_WARNINGS', '1'))` discussed as an alternative? Seems to me that the behavior could be entirely be replaced with pytest.

Reply:
Yeah, that might work. There are places though were it is still nice to have the test run even without the strict warning test.

These cleanups have a hard time getting through the review process. so i doubt anything sweeping will get done.

Reply:
I guess there are more pressing matters right now. :+1:

Reply:
I've been in favour of fully embracing pytest, but it's still pretty contentious so let's leave it for another time...